### PR TITLE
Add parsing for CSSRotation

### PR DESCRIPTION
**Pull request by wilddamon**

### CODE REVIEW

File: test/js/parsing.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    assert.deepEqual(consumeNumber('-3.4e-2 123'), [-3.4e-2, ' 123']);
+  });
+
+  test('consumeRepeated with separator', function() {
+    var consumeRepeated = typedOM.internal.parsing.consumeRepeated;
+    var consumer = typedOM.internal.parsing.consumeNumber;
+    assert.deepEqual(consumeRepeated(consumer, /^,/, '1,2,3'), [[1,2,3], '']);
+    assert.deepEqual(consumeRepeated(consumer, /^,/, '1,2,3,'), [[1,2,3], '']);
+    assert.deepEqual(consumeRepeated(consumer, /^,/, '1,2,3,a'), [[1,2,3], 'a']);
+    assert.deepEqual(consumeRepeated(consumer, /^,/, '  1 ,   2 ,  3    '), [[1,2,3], '']);
```
Comment (rjwright) [inline test/js/parsing.js:53]: Others that might be worth testing:
`1, 2 3, 4`
`1, 2, a, 3`

Maybe also test that the consumer is matched and then the separator? e.g.
`assert.deepEqual(consumeRepeated(consumer, /^\./, '1.0.2.0.3.0'), [[1.0,2.0,3.0], ''])`

File: test/js/parsing.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+  test('consumeRepeated with null (space) separator', function() {
+    var consumeRepeated = typedOM.internal.parsing.consumeRepeated;
+    var consumer = typedOM.internal.parsing.consumeNumber;
+    assert.deepEqual(consumeRepeated(consumer, null, '1 2 3'), [[1,2,3], '']);
+    assert.deepEqual(consumeRepeated(consumer, null, '1 2 3 a'), [[1,2,3], 'a']);
+    assert.deepEqual(consumeRepeated(consumer, null, '  1    2   3    '), [[1,2,3], '']);
+  });
+
+  test('consumeRepeated with max results', function() {
```
Comment (rjwright) [inline test/js/parsing.js]: "with max results and separator"

File: test/js/parsing.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    var consumeRepeated = typedOM.internal.parsing.consumeRepeated;
+    var consumer = typedOM.internal.parsing.consumeNumber;
+    assert.deepEqual(consumeRepeated(consumer, /^,/, '1,2,3', 2 /* opt_max */), [[1,2], '3']);
+    assert.deepEqual(consumeRepeated(consumer, /^,/, '1,2,3,', 2), [[1,2], '3,']);
+    assert.deepEqual(consumeRepeated(consumer, /^,/, '1,2,3,a', 2), [[1,2], '3,a']);
+    assert.deepEqual(consumeRepeated(consumer, /^,/, '  1 ,   2 ,  3    ', 2), [[1,2], '  3    ']);
+  });
+
+  test('consumeRepeated with max results and separator', function() {
```
Comment (rjwright) [inline test/js/parsing.js]: "with max results and null (space) separator"

**Comment by rjwright** [comment]
LGTM with nits

File: test/js/parsing.js (hunk 1 above)
Comment (wilddamon) [inline test/js/parsing.js:53]: Done.

File: test/js/parsing.js (hunk 2 above)
Comment (wilddamon) [inline test/js/parsing.js]: Done.

File: test/js/parsing.js (hunk 3 above)
Comment (wilddamon) [inline test/js/parsing.js]: Done

**Comment by rjwright** [comment]
Merge if you're ready :)

**Pull request merged**